diarize_parallel.py

```diff
@@ -53,13 +61,15 @@
 else:
     vocal_target = args.audio
 
+print("Starting Nemo process with vocal_target: ", vocal_target)
```

Comment:
using `logging` is preferred

Reply:
I've replaced all print's with logging now